src/content_scripts/initialize.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Initialize various pieces of the content script
+ */
+"use strict";
+
+/* eslint-disable no-unused-vars */
+
+/* globals initializeUpdateRequest, initializeApplyColorSchemeCss, initializeApplyColorSchemeJs */
+/* globals functionalMode:writable, fakedColorStatus:writable, COLOR_STATUS */
+
+// initial settings at document-start, will be filled by a dynamic content script
+// eslint-disable-next-line no-var
```

Comment:
Please add an explanation why one needs to use var here.

Reply:
Would https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#temporal_dead_zone_tdz do?